Modules/BenchPress.Azure/Public/Get-ResourceByType.ps1

```diff
@@ -43,29 +43,7 @@ function Get-ResourceByType {
       The name of the Resource Group
 
     .PARAMETER ResourceType
-      The type of the Resource (currently support the following:
-      ActionGroup
-      AksCluster
-      AppInsights
-      AppServicePlan
-      ContainerRegistry
-      DataFactory
-      DataFactoryLinkedService
-      EventHub
-      EventHubConsumerGroup
-      EventHubNamespace
-      KeyVault
-      ResourceGroup
-      SqlDatabase
-      SqlServer
-      StorageAccount
-      StorageContainer
-      StreamAnalyticsCluster
-      SynapseSparkPool
-      SynapseSqlPool
-      SynapseWorkspace
-      VirtualMachine
-      WebApp)
+      The type of the Resource.
```

Comment:
Is there a reason we deleted this? Something from the design reviews with the expers?

Reply:
It was originally added because the parameter was converted to a string from an enum. This documented the valid values that could be passed as a string. Since this was converted back to an enum, this is no longer necessary because the enum itself describes the possible values.